docs/rules/error-message.md

```diff
@@ -0,0 +1,43 @@
+# Enforce passing a `message` value when throwing an error
+
+This rule enforces a `message` value to be passed in when throwing an instance of an `Error` object, which leads to more readable code.
+
+
+## Fail
+
+```js
+throw Error();
+```
+
+```js
+throw Error('');
+```
+
+```js
+throw new TypeError();
+```
+
+```js
+throw new MyError();
```

Comment:
Per https://github.com/sindresorhus/eslint-plugin-unicorn/pull/140#issuecomment-355846567, remove this example.